Solution1/ConsoleApp/Configuration/FileConfig.cs

```diff
@@ -5,12 +5,61 @@ namespace ConsoleApp.Configuration
 {
     public class FileConfig : IConfig
     {
-        public int MinCountDaysForecast => int.TryParse(ConfigurationManager.AppSettings["minCountDays"], out var value) ? value : default;
+        public AppConfiguration AppConfig { get => new AppConfiguration(); }
 
-        public int MaxCountDaysForecast => int.TryParse(ConfigurationManager.AppSettings["maxCountDays"], out var value) ? value : default;
+        public WeatherApiConfiguration ApiConfig { get => new WeatherApiConfiguration(); }
```

Comment:
is it necessary to do it? if there is no weather config it should be error, isn't it?